src/main/java/lotto/model/WinningLotto.java

```diff
@@ -1,24 +1,54 @@
 package lotto.model;
 
+import lotto.prize.LottoPrize;
+import lotto.prize.MatchInformation;
+
+import java.util.List;
 import java.util.Objects;
+import java.util.Set;
+
+import static java.util.stream.Collectors.toList;
 
 public class WinningLotto {
-    private final Lotto winningLotto;
+    private final Set<LottoNumber> winningNumber;
```

Comment:
```suggestion
    private final Lotto winningLotto;
```
이렇게 Lotto 객체를 가지고 있는게 자연스럽지 않을까요?